examples/core/graphsage/node_classification.py

```diff
@@ -99,7 +99,7 @@ def train(g, features, labels, masks, model):
     #
     # First, the whole graph is loaded and transformed. Then the training
     # process is performed on a model which is composed of 2 GraphSAGE-gcn
-    # layer. Finally, the performance of the model is evaluated on test set.
+    # layers. Finally, the performance of the model is evaluated on test set.
```

Comment:
please do not include unrelated change such as typo fix. it's better to do in a separate PR.